input/community/index.md

```diff
@@ -2,6 +2,6 @@ Title: Links to the Axiom3D Communities
 ---
 
 - [Join the Slack Community!](join-slack)
+<div><img style="width: 20%;" src="/assets/img/slacklogo.png" alt=""/></div>
 
-Facebook
```

Comment:
Please revert, both twtter and facebook exist, just need to add the links.

Reply:
Facebook ouch :)

Reply:
Actually, no facebook, but gitter